src/Craftify.Geometry.Domain/LineSegmentsGraphAlgorithms.cs

```diff
@@ -0,0 +1,135 @@
+﻿namespace Craftify.Geometry.Domain;
+
+public static class LineSegmentsGraphAlgorithms
+{
+    public static IReadOnlyList<LineSegment> Dijkstra(
```

Comment:
Why do we name the method after the algorithm? Are we focusing on the algorithm or the functionality it provides? I think a name like `FindShortestPath` tells me more about what it does. The client shouldn't be concerned with the algorithm used.